framework/test-vectors/complete-vectors/kms-mrk-aware-discovery.md

```diff
@@ -0,0 +1,23 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Aws Kms Mrk Discovery Keyring Vectors
+
+## Version
+
+1.0.0
+
+## Summary
+
+This describes the test cases for the [Aws Kms Mrk Discovery Keyring](../../aws-kms/aws-kms-mrk-discovery-keyring.md)
+
+## Reference-level Explanation
+
+### Basic tests
+
+A test MUST attempt to encrypt with a single region key.
+A test MUST attempt to encrypt with a multi region key.
```

Comment:
Negative Tests:
- Encrypt/Decrypt with MRK not replicated to region
- Encrypt/Decrypt with Filter excluding all replicated regions